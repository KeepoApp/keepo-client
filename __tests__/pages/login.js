import React from 'react';
import renderer from 'react-test-renderer';

import Login from '../../app/containers/pages/login/index';

describe('Login page', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <Login />,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
