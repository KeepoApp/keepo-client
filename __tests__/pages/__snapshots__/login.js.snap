// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page should render correctly 1`] = `
<div
  className="login flex-center"
>
  <div
    className="login__box"
  >
    <div
      className="box__logo flex-center"
    >
      <img
        alt=""
        src="/assets/img/logo.png"
      />
      <div
        className="divider"
      />
    </div>
    <div
      className="box__social"
    >
      <h2>
        Login
      </h2>
      <p>
        Give thanks to the most high. Stay focused. Always remember in the jungle thereâ€™s a lot of they in there.
      </p>
      <button
        onClick={[Function]}
      >
        Login with Google
      </button>
    </div>
  </div>
  <div
    className="login__love"
  >
    <p>
      Made with 
      <i
        className="fa fa-heart"
      />
       by Wesley de Bruijn and Jeffrey van Hoven
    </p>
  </div>
</div>
`;
